src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -15,12 +15,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- This property disables the conflict resolution logic from the Microsoft.Packaging.Tools package, which is superceded by the logic here in the SDK -->
     <DisableHandlePackageFileConflicts>true</DisableHandlePackageFileConflicts>
   </PropertyGroup>
-
+  
```

Comment:
nit: can you remove the extra space?

Reply:
Yes, thanks for the call out. 

I installed this extension now https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave&ssr=false#overview so these artifacts of removed changes shouldn't reoccur. Sadly there were some problems elsewhere in the file that also got changed. Not sure if that's going to be annoying in the future if my auto-formatter is fixing the format of code that's in the same file but not relevant to my changes? (IMO it's better to fix the formatting everywhere.)